test/contracts-network/token-locking.js

```diff
@@ -118,22 +132,12 @@ contract("Token Locking", (addresses) => {
 
     it("should be able to deposit tokens multiple times if they are unlocked", async () => {
       await token.approve(tokenLocking.address, usersTokens, { from: userAddress });
-      await tokenLocking.deposit(token.address, usersTokens / 2, { from: userAddress });
-      await tokenLocking.deposit(token.address, usersTokens / 2, { from: userAddress });
+      await tokenLocking.methods["deposit(address,uint256,bool)"](token.address, usersTokens / 2, true, { from: userAddress });
+      await tokenLocking.methods["deposit(address,uint256,bool)"](token.address, usersTokens / 2, true, { from: userAddress });
       const info = await tokenLocking.getUserLock(token.address, userAddress);
       expect(info.balance).to.eq.BN(usersTokens);
     });
 
-    it("should be able to deposit tokens while they are locked, into pendingBalance", async () => {
```

Comment:
This test should be turned in to `should not be able to deposit tokens while they are locked`, rather than being deleted.